### PR TITLE
Validationerror

### PR DESCRIPTION
**Pull request by sufyaniqbal**
# Description
change to allow validation to work again

# How to test?
env: spp2824 
trigger validations

# Links
https://collaborate2.ons.gov.uk/jira/browse/SPP-3928

### CODE REVIEW

**Review by mattprice-28 — APPROVED** [review]

File: app/templates/view_form/BaseFormView.html [hunk 1]
```diff
@@ -94,9 +94,9 @@ <h2 class="tabs__title">Tabs</h2>
 
                 <ul class="tabs__list tabs__list--row">
                     <li id="tabId1Item" class="tab__list-item">
-                        <a href="#tabId1" class="tab tab--row" data-ga="click" data-ga-category="tabs" data-ga-action="Show: Current response" data-ga-label="Show: Current response" aria-selected="true">Current response</a></li>
+                        <a href="#tabId1" class="tab tab--row" data-ga="click" data-ga-category="tabs" data-ga-action="Show: Current Data" data-ga-label="Show: Current Data" aria-selected="true">Current response</a></li>
```
Comment (rhiGriff) [inline app/templates/view_form/BaseFormView.html:97]: Why have these been changed back to data? These were changed to response as per story SPP-830: https://collaborate2.ons.gov.uk/jira/browse/SPP-830

File: app/templates/view_form/BaseFormView.html [hunk 2]
```diff
@@ -94,9 +94,9 @@ <h2 class="tabs__title">Tabs</h2>
...
                     <li id="tabId2Item" class="tab__list-item">
-                        <a href="#tabId2" class="tab tab--row" data-ga="click" data-ga-category="tabs" data-ga-action="Show: Historical responses" data-ga-label="Show: Historical responses" aria-selected="false">Historical responses</a></li>
+                        <a href="#tabId2" class="tab tab--row" data-ga="click" data-ga-category="tabs" data-ga-action="Show: Historic Data" data-ga-label="Show: Historic Data" aria-selected="false">Historical responses</a></li>
```
Comment (rhiGriff) [inline app/templates/view_form/BaseFormView.html:99]: Why have these been changed back to data? These were changed to responses as per story SPP-830: https://collaborate2.ons.gov.uk/jira/browse/SPP-830

**Review by rhiGriff — APPROVED** [review]
2 questions around why this has changed

**Review by rhiGriff — CHANGES_REQUESTED** [review]
Questions around change

**Review by rhiGriff — APPROVED** [review]
Reasonings make sense as lon as display to users useres response / responses all good

**Pull request merged**